src/site/xdoc/server/config-smtp-lmtp.xml

```diff
@@ -152,8 +152,8 @@
       
         <li>As mentioned above, SMTP AUTH requires that Apache James Server be able to distinguish between mail intended 
             for local delivery and mail intended for remote delivery.  Apache James Server makes this determination by matching the 
-            domain to which the mail was sent against the &lt;servernames&gt; element of the Apache James Server configuration block.  Any 
-            local domains should be explicitly listed as &lt;servername&gt; elements in this section.</li>
+            domain to which the mail was sent against the &lt;DomainList&gt; component, configured in <a href="/server/config-domainlist.xml">domainlist.xml</a> configuration file. Any 
+            local domains should be explicitly listed as &lt;domainname&gt; elements in this section.</li>
```

Comment:
as far as I know, DomainList can be implemented with a database (like postgres with JPA or cassandra), so the wording may be misleading here.

Reply:
yes, probably it should just point to the domain-list documentation page without stating about domainname elements which are implementation specific.